alcs-frontend/src/app/services/meeting/meeting.service.ts

```diff
@@ -0,0 +1,24 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { firstValueFrom } from 'rxjs';
+import { environment } from '../../../environments/environment';
+import { ToastService } from '../toast/toast.service';
+import { UpcomingMeetingBoardMapDto } from './meeting.dto';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class MeetingService {
```

Comment:
If it is specific to Decision I would add that into the name

Reply:
Done, renamed to decision-meeting.service.ts